lib/Pod/To/Text.pm6

```diff
@@ -33,14 +33,32 @@ sub pod2text($pod) is export {
 
 sub heading2text($pod) {
     given $pod.level {
-        when 1  {          pod2text($pod.contents)  }
-        when 2  { '  '   ~ pod2text($pod.contents)  }
+        my $content = $pod.contents.first.contents.grep( * !~~ "").first;
+        my $text;
+        if $content ~~ Str {
+            $text = $content;
+        } else {
+            $text = pod2text $content;
+        }
+        my $name = $text.lines[0];
+        my $rest = $text.lines[1…*].join: "\n";
```

Comment:
Please change `[1…*]` at least to `[1..*]`, using three dots is 3x+ slower since it creates a sequence not a range.